Index: ~/.vim/plugged/vim-signify/autoload/sy/highlight.vim
--- ~/.vim/plugged/vim-signify/autoload/sy/highlight.vim
+++ ~/.vim/plugged/vim-signify/autoload/sy/highlight.vim
@@ -5,14 +5,14 @@ scriptencoding utf-8
 " Init: values {{{1
 if get(g:, 'signify_sign_show_text', 1)
   let s:sign_add               = get(g:, 'signify_sign_add',               '+')
-  let s:sign_delete_first_line = get(g:, 'signify_sign_delete_first_line', '[7m<80>)
+  let s:sign_delete_first_line = get(g:, 'signify_sign_delete_first_line', '-')
   let s:sign_change            = get(g:, 'signify_sign_change',            '!')
   let s:sign_changedelete      = get(g:, 'signify_sign_changedelete',      s:sign_change)
 else
-  let s:sign_add               = ' '
-  let s:sign_delete_first_line = ' '
-  let s:sign_change            = ' '
-  let s:sign_changedelete      = ' '
+  let s:sign_add               = '+'
+  let s:sign_delete_first_line = '-'
+  let s:sign_change            = '!'
+  let s:sign_changedelete      = '!'
 endif
 
 let s:sign_show_count = get(g:, 'signify_sign_show_count', 1)
