Index: ~/.vim/plugged/nerdtree/plugin/NERD_tree.vim
--- ~/.vim/plugged/nerdtree/plugin/NERD_tree.vim
+++ ~/.vim/plugged/nerdtree/plugin/NERD_tree.vim
@@ -70,8 +70,8 @@ call s:initVariable("g:NERDTreeShowLineNumbers", 0)
 call s:initVariable("g:NERDTreeSortDirs", 1)
 
 if !nerdtree#runningWindows() && !nerdtree#runningCygwin()
-    call s:initVariable("g:NERDTreeDirArrowExpandable", "[7m<96>)
-    call s:initVariable("g:NERDTreeDirArrowCollapsible", "[7m<96>)
+    call s:initVariable("g:NERDTreeDirArrowExpandable", "+")
+    call s:initVariable("g:NERDTreeDirArrowCollapsible", "~")
 else
     call s:initVariable("g:NERDTreeDirArrowExpandable", "+")
     call s:initVariable("g:NERDTreeDirArrowCollapsible", "~")
